airflow/utils/decorators.py

```diff
@@ -87,6 +87,23 @@ def wrapper(*args, **kwargs):
         return result
     return wrapper
 
+def export_execute_debug_metadata(func):
+    """
+    This decorator can be used on an operator's execute function in order to export various
+    dag/task id/run information to the environment for debugging/monitoring purposes.
+    """
+    def wrapped(self, *args, **kwargs):
+        if self.dag:
+            os.environ['AIRFLOW_AIRBNB_DAG_ID'] = self.dag.dag_id
+            if self.dag.start_date:
+                os.environ['AIRFLOW_AIRBNB_DAG_RUN_ID'] = self.dag.start_date.isoformat()
```

Comment:
DAG_RUN_ID is being set to the global start date of the DAG here which I don't think is what you want based on the variable name.